capillary-services/PARTNER-PRICE.md

```diff
@@ -0,0 +1,441 @@
+# Capillary Services Endpoints
+
+## Website -Price and Stock -API
+
+### Fields/Data Model
+
+
+**uuid**
+
+- Data type: uuid, Must
+- Description: Product ID, could be product_uuid or byprice_uuid
+- Validation: Should be a proper longitude
+- Examples: f6bafc68-06b3-4c09-b044-978a0a7404f6 or 1d73aa2f-beee-426f-bbcd-f69556aaf873
+
+**universal_product_id**
+
+- Data type: ean or gtin, Must
+- Description: Universal Product ID, could be ean or gtin
+- Validation: Should be a proper longitude
+- Examples: 7501409201041 or 7501299330265
+
+**latitude**
+
+- Data type: float, Must
+- Description: Latitude of center point
+- Validation: Should be a proper latitude
+- Examples: 19.308533 or 19.336679
+
+**longitude**
+
+- Data type: float, Must
+- Description: Longitude of center point
+- Validation: Should be a proper longitude
+- Examples: -99.125781 or -99.112842
+
+**radius**
+
+- Data type: int, optional but strongly recommended
+- Description: Radius of area to get prices from
+- Validation: Should be an integer from 1 to 100
+- Examples: 10 or 30
```

Comment:
Why 30 when you know our max limit is 10

Reply:
changed